test/metabase/pulse_test.clj

```diff
@@ -866,9 +866,9 @@
                                                  :channel_type "slack"
                                                  :details      {:channel "#general"}}]
           (pulse.test-util/slack-test-setup
-           (let [pulse-data (metabase.pulse/send-pulse! (pulse/retrieve-pulse pulse-id))
-                 slack-data (m/find-first map? pulse-data)
-                 email-data (first (m/find-first seq? pulse-data))]
+           (let [pulse-data (flatten (metabase.pulse/send-pulse! (pulse/retrieve-pulse pulse-id)))
```

Comment:
Flatten is ambiguous to readers and dangerous to refactors, how about `(into [] cat ...)` instead? 

Or `(apply concat ...)` if you don't want to be cool 😎